scripts/add_extra_components.py

```diff
@@ -187,14 +187,17 @@ def attach_stores(n, costs, config):
     if ("csp" in config["renewable"].keys()) and (
```

Comment:
Hi @FabianHofmann, Just calling your attention to a recent bug here. 

There is a recent discussion ongoing here about it [here](https://github.com/pypsa-meets-earth/pypsa-earth/pull/1076/files/912809dff3b80fb92cea2a353f3f52018bdd175a)